common/flagdproxy/flagdproxy.go

```diff
@@ -206,3 +223,14 @@ func (f *FlagdProxyHandler) getOwnerReference(ctx context.Context) (metav1.Owner
 	}, nil
 
 }
+
+func (f *FlagdProxyHandler) getOwnerReferences(ctx context.Context) []metav1.OwnerReference {
```

Comment:
can't we combine this functionality with `getOwnerReference(ctx context.Context) (metav1.OwnerReference, error)` ?